src/probnum/randprocs/markov/_markov.py

```diff
@@ -69,6 +46,23 @@ def mean(self) -> _function.Function:
     def cov(self) -> "MarkovProcess.Kernel":
         return MarkovProcess.Kernel(self.__call__)
 
+    class Kernel(kernels.Kernel):
```

Comment:
the codecov complaints from here on below are not the fault of this PR, because it simply moves the objects around. So I would propose to ignore them.